src/MountedTraversal.js

```diff
@@ -257,7 +257,7 @@ export function pathToNode(node, root) {
 }
 
 export function parentsOfInst(inst, root) {
-  return pathToNode(inst, root).reverse();
+  return (pathToNode(inst, root) || []).reverse();
```

Comment:
When I change that, another test breaks that I can't figure out. I'll push that commit up so you can play with it.